performance/patient-encounter-observation-bundle-create.js

```diff
@@ -0,0 +1,102 @@
+/* global open __ENV */
+/* eslint no-undef: "error" */
+
+import http from 'k6/http'
+import { check } from 'k6'
+
+const patientResource = JSON.parse(open('./resources/patient.json'))
+const encounterResource = JSON.parse(open('./resources/encounter.json'))
+const heightObservation = JSON.parse(open('./resources/heightResource.json'))
+const weightObservation = JSON.parse(open('./resources/weightResource.json'))
+
+const BASE_URL = __ENV.BASE_URL || 'http://localhost:3447'
+
+export const options = {
+  vus: 10,
+  iterations: 1000,
```

Comment:
Minor: Lets update this to 10000 instead to give hearth a bit longer time to process and consume resources. 1000 is being processed very quickly (5 seconds :) )

Reply:
This comment is no longer valid as we made this a time duration tests to see how many bundles can be processed during this time